docs/docs/sticky-bucketing.mdx

```diff
@@ -30,16 +30,34 @@ Sticky Bucketing is a GrowthBook Pro and Enterprise feature.
 
 :::
 
-## Setting up Sticky Bucketing
+## Setting up Sticky Bucketing with the Script Tag SDK
+
+### 1. Configure the script tag
+
+Using Sticky Bucketing with our [HTML Script Tag](https://docs.growthbook.io/lib/script-tag) is simple. Simply add either of these properties to your script tag:
+
+- `data-use-sticky-bucket-service="cookie"` or
```

Comment:
Not sure if it'd be helpful to put the full html example. My thinking here is that there may be low/no-code users copy/pasting:

Store Sticky Bucket assignments in cookies:
```html
<script defer
  data-client-key="YOUR_CLIENT_KEY_HERE"
// highlight-next-line
  data-use-sticky-bucket-service="cookie"
  src="https://cdn.jsdelivr.net/npm/@growthbook/growthbook/dist/bundles/auto.min.js"
></script>
```
...